tests/std/tests/GH_005402_string_with_volatile_range/test.cpp

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#if _HAS_CXX23
+#include <ranges>
+#endif // _HAS_CXX23

```

Comment:
Nitpick: This interferes with sorting and we should group it separately.

(There are a number of pre-existing cases of this that I'll get around to cleaning up someday)